book/0 - Introduction/1 - Introduction.html

```diff
@@ -22,9 +22,10 @@
 <ul>
 	<li><a href="http://cuberite.org/">Cuberite Homepage</a></li>
 	<li><a href="https://forum.cuberite.org/">Forum</a></li>
-	<li><a href="https://forum.cuberite.org/forum-2.html">Plugin Repository</a></li>
+	<li><a href="https://cuberite.org/plugins/">Plugin Repository</a></li>
```

Comment:
I disagree.

1. The old repo should be linked to. It has the majority of plugins and it is more mature.
2. Installation guide should be linked to.

In my opinion the ideal option is to have both https://forum.cuberite.org/forum-2.html and https://plugins.cuberite.org mentioned in https://cuberite.org/plugins/